store/src/genesis.rs

```diff
@@ -9,10 +9,12 @@ use miden_objects::{
 };
 use once_cell::sync::Lazy;
 
-use crate::config::{APP, ORG};
+use crate::{
+    config::{APP, ORG},
+    state::ACCOUNT_DB_DEPTH,
+};
 
-// FIXME: This is a duplicate of the constant in `store::state`
-pub(crate) const ACCOUNT_DB_DEPTH: u8 = 64;
+pub const GENESIS_HEADER_BLOCK_NUM: u32 = 0;
```

Comment:
nit: I'd probably call this just `GENESIS_BLOCK_NUM`.

Reply:
done